{"name": "Test of failed attempt to create an order without authorization", "status": "failed", "statusDetails": {"message": "AssertionError: Instead of code 401 received code 200\nassert 200 == 401\n +  where 200 = <Response [200]>.status_code", "trace": "self = <tests.test_order_creation.TestOrderCreation object at 0x0000026D49C8C290>\n\n    @allure.title('Test of failed attempt to create an order without authorization')\n    @allure.description('Test of the endpoint \"Создание заказа\" POST /api/orders.'\n                        'Checks that order creation is impossible without authorization '\n                        'that response body contains all main required fields,'\n                        'and that request returns \"success\": False')\n    def test_order_creation_without_authorization_failed(self):\n        payload = {\"ingredients\": [\"61c0c5a71d1f82001bdaaa6d\"]}\n        response = requests.post(CommonData.create_order_url, data=payload)\n        \"\"\"Согласно документации, логике работы сайта и здравому смыслу,\n        неавторизованный пользователь не может оформить заказ\n        (раздел \"Авторизация и регистрация\", стр 5 - \"Только авторизованные пользователи могут делать заказы\").\n        Так как в документации нет примера ответа для такого случая, то в качестве образца принят\n        ответ на запрос изменения данных пользователя:\n        \"Если выполнить запрос без авторизации, вернётся код ответа 401 Unauthorized.\n        {\"success\": false, \"message\": \"You should be authorised\"}\"\"\"\n>       assert response.status_code == 401, f'Instead of code 401 received code {response.status_code}'\nE       AssertionError: Instead of code 401 received code 200\nE       assert 200 == 401\nE        +  where 200 = <Response [200]>.status_code\n\ntests\\test_order_creation.py:40: AssertionError"}, "description": "Test of the endpoint \"Создание заказа\" POST /api/orders.Checks that order creation is impossible without authorization that response body contains all main required fields,and that request returns \"success\": False", "start": 1701420711771, "stop": 1701420713208, "uuid": "a3520505-9bc7-4300-b7f0-1251cd0387a5", "historyId": "bdd7ec42b810dddcd700106118a0da1d", "testCaseId": "bdd7ec42b810dddcd700106118a0da1d", "fullName": "tests.test_order_creation.TestOrderCreation#test_order_creation_without_authorization_failed", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_order_creation"}, {"name": "subSuite", "value": "TestOrderCreation"}, {"name": "host", "value": "Airforce3"}, {"name": "thread", "value": "10084-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_order_creation"}]}